azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/functions/VMImageToImage.java

```diff
@@ -77,15 +73,12 @@ public static String encodeFieldsToUniqueId(VMImage imageReference){
 
    @Override
    public Image apply(final VMImage image) {
-
-      Credentials credentials = new Credentials(AZURE_LOGIN_USERNAME, AZURE_LOGIN_PASSWORD);
       final ImageBuilder builder = new ImageBuilder()
               .name(image.offer)
               .description(image.sku)
               .status(Image.Status.AVAILABLE)
               .version(image.sku)
               .id(encodeFieldsToUniqueId(image))
-              .defaultCredentials(LoginCredentials.fromCredentials(credentials))
```

Comment:
As commented before, there's no need to do this here. jclouds already decorates all images with the default credentials [here](https://github.com/jclouds/jclouds/blob/master/compute/src/main/java/org/jclouds/compute/config/ComputeServiceAdapterContextModule.java#L121).